egs/wsj/s5/utils/validate_lang.pl

```diff
@@ -131,6 +140,13 @@ sub check_allowed_whitespace {
 $lang = shift @ARGV;
 $exit = 0;
 $warning = 0;
+
+if (!$subword_check) {
+  $word_boundary = "word_boundary";
```

Comment:
I think it would be better to detect this automatically rather than relying on a flag; this script is called from many scripts without args.

Reply:
Can I write the separator into file data/lang/separator and check the existence of this file during validation?